specification/security/resource-manager/Microsoft.Security/preview/2022-08-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1318 @@
+      ],
+      "properties": {}
+    },
+    "cspmMonitorAzureDevOpsOffering": {
+      "type": "object",
+      "description": "The CSPM monitoring for AzureDevOps offering",
+      "x-ms-discriminator-value": "CspmMonitorAzureDevOps",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
+    },
+    "defenderCspmAwsOffering": {
+      "type": "object",
+      "description": "The CSPM P1 for Aws offering",
+      "x-ms-discriminator-value": "DefenderCspmAws",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "vmScanners": {
+          "type": "object",
+          "description": "The Microsoft Defender for Server VM scanning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Microsoft Defender for Server VM scanning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Microsoft Defender for Server VM scanning",
+              "properties": {
+                "cloudRoleArn": {
+                  "type": "string",
+                  "description": "The cloud role ARN in AWS for this feature"
+                },
+                "scanningMode": {
+                  "type": "string",
+                  "description": "The scanning mode for the vm scan.",
+                  "enum": [
+                    "Default"
+                  ],
+                  "x-ms-enum": {
+                    "name": "scanningMode",
+                    "modelAsString": true
+                  }
+                },
+                "exclusionTags": {
+                  "type": "object",
+                  "description": "VM tags that indicates that VM should not be scanned",
+                  "properties": {}
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "defenderCspmGcpOffering": {
+      "type": "object",
+      "description": "The CSPM P1 for GCP offering",
+      "x-ms-discriminator-value": "DefenderCspmGcp",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
+    },
+    "defenderForDevOpsGithubOffering": {
+      "type": "object",
+      "description": "The Defender for DevOps for Github offering",
+      "x-ms-discriminator-value": "DefenderForDevOpsGithub",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
+    },
+    "defenderForDevOpsAzureDevOpsOffering": {
+      "type": "object",
+      "description": "The Defender for DevOps for Azure DevOps offering",
+      "x-ms-discriminator-value": "DefenderForDevOpsAzureDevOps",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {}
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

provide definition

Reply:
properties are meant to allow customers to provide additional information for the offering enablement 
in this case no additional properties are required for the offering enablement. that's why properties are empty.